package.json

```diff
@@ -34,6 +34,7 @@
   },
   "dependencies": {
     "@wikia/ad-engine": "git+https://github.com/Wikia/ad-engine.git#v23.4.0",
+    "@wikia/search-tracking": "github:Wikia/search-tracking#adjustments",
```

Comment:
change it to master / release tag when `adjustments` branch is merged / released